modules/containers-signature-verify-enable.adoc

```diff
@@ -94,3 +95,91 @@ $ butane 51-master-rh-registry-trust.bu -o 51-master-rh-registry-trust.yaml
 ----
```

Comment:
I think we need delete line 81~89, because the master node is enabled image signature verification by default. We don't need add machine config to realize it.